src/components/templates/product-reviews-table/index.tsx

```diff
@@ -0,0 +1,87 @@
+import moment from "moment"
+import React from "react"
+import { ProductReview } from "../../../services/nibll-api"
+import Table, { TablePagination } from "../../molecules/table"
+
+type ProductReviewsTableProps = {
+  reviews: ProductReview[]
+  pageNumber: number
+  pageSize: number
+  totalResults: number
+  filter?: { product_variant_id?: number | number[] }
+  sort?: { column?: string; direction?: "asc" | "desc" }
+  setPageNumber: (number: number) => void
+}
+
+const ProductReviewsTable: React.FC<ProductReviewsTableProps> = ({
+  reviews,
+  pageNumber,
+  pageSize,
+  totalResults,
+  setPageNumber,
+}) => {
+  console.log(pageNumber, pageSize, totalResults)
+
+  const getRow = (review, index) => {
```

Comment:
@dwene this is kind of a pattern that I removed out of CLX a long time ago. Technically instead of a function, this would be better of abstracted as a ReviewRow component

then implemented as

```
{reviews.map((review, index) => <ReviewRow key={review.id} review={review} index={index} /> )}
```

Reply:
Ah yes... I was just copying their pattern, I like that better.